common/extensions/api/brave_rewards.json

```diff
@@ -1151,6 +1134,43 @@
           }
         ]
       },
+      {
+        "name": "getAdsStatement",
+        "type": "function",
+        "description": "Get ads statement",
+        "parameters": [
+          {
+            "type": "function",
+            "name": "callback",
+            "parameters": [
+              {
+                "name": "adsStatement",
+                "type": "object",
+                "properties": {
+                  "success": {
+                    "type": "boolean"
+                  },
+                  "estimatedPendingRewards": {
+                    "type": "number"
+                  },
+                  "nextPaymentDate": {
+                    "type": "string"
+                  },
+                  "adsReceivedThisMonth": {
+                    "type": "integer"
```

Comment:
Other types in this object are `number` but this one is `integer`. I'm not really sure which is preferred (maybe check other object definitions in this file), but probably good to remain consistent.

Reply:
see `onPromotionFinish` which has mixed types too. I am using `integer` for non floating point types and `number` for floating point types.